src/Makefile

```diff
@@ -2,10 +2,13 @@
 # Use `FC=<compiler> ... make` to select compiler
 # Use `DEBUG=1 ... make` for a debug build
 # Use `NETCDF=1 ... make` to build with NetCDF using `nf-config`
+# compile with either FC=ifort or FC=gfortran
```

Comment:
```suggestion
```
Line 2 already describes `FC`

Reply:
I agree with Zach here, as line 2 is self explanatory.  Doesn't need to be stated again.